packages/server/server/common/passport/passportUtils.ts

```diff
@@ -57,9 +59,18 @@ export function signJwt(user: User): Promise<string> {
     };
 
     return new Promise<string>((resolve, reject) => {
-        Jwt.sign(JwtPayload, process.env.PRIVATE_KEY, JwtOptions, (err, token) => {
-            if (err) return reject(err);
-            resolve(token);
-        });
+        Jwt.sign(
+            JwtPayload,
+            process.env.PRIVATE_KEY.replace(/\\n/g, '\n'),
```

Comment:
this must be annoying to deal with 😅

Reply:
Nah this one was still ok, the logging gave pretty descriptive errors. I was stuck more last time i tried deploying (with the catch all * for app.get hahaha)